src/Storages/RabbitMQ/StorageRabbitMQ.h

```diff
@@ -99,9 +112,13 @@ class StorageRabbitMQ final: public ext::shared_ptr_helper<StorageRabbitMQ>, pub
     std::mutex mutex;
     std::vector<ConsumerBufferPtr> buffers; /// available buffers for RabbitMQ consumers
 
-    size_t next_channel_id = 1; /// Must >= 1 because it is used as a binding key, which has to be > 0
-    bool update_channel_id = false;
-    std::atomic<bool> loop_started = false;
+    String unique_strbase;
+    String sharding_exchange, bridge_exchange, consumer_exchange;
+    std::once_flag flag;
+    size_t consumer_id = 0;
+    std::atomic<size_t> producer_id = 1;
```

Comment:
Need to add a comment for each flag (in all classes). It's extremely hard to read a code where we have so complicated state which can be changed for different threads. What does it mean when it's set to true? To false? If counter, than what we count?